subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/junit/JUnitOptions.java

```diff
@@ -31,6 +32,19 @@ public class JUnitOptions extends TestFrameworkOptions {
 
     private Set<String> excludeCategories = new LinkedHashSet<String>();
 
+    public JUnitOptions() {
+    }
+
+    /**
+     * Copies the JUnit options.
+     * @since 8.0
+     */
+    @Incubating
```

Comment:
I've added both `@Incubating` and the `@since 8.0` annotation in order to make the Binary compatibility report happy. I guess the `@since` definitely makes sense, but I'm not sure we should keep the `@Incubating` here? I mean it's not like a new feature in preview mode...

Reply:
DONE (64808b88)